kubernetes_deploy/staging/deployment.yml

```diff
@@ -36,7 +36,7 @@ spec:
         - name: GOOGLE_MAPS_API_KEY
           value: AIzaSyDdz1B1M2-2r0CQKJPscvmNEyxGwSZkXJk
         - name: LAALAA_API_HOST
-          value: https://laa-legal-adviser-api-staging.apps.live-1.cloud-platform.service.justice.gov.uk
+          value: https://staging.laalaa.dsd.io
```

Comment:
I think all-in-all, the Kubernetes staging environment is still more stable than the template-deploy one and once we upload to it the data will stay there for testing purposes.

Could we remove this change, please?